graphql_compiler/cost_estimation/analysis.py

```diff
@@ -408,6 +408,19 @@ def types(self) -> Dict[VertexPath, Union[GraphQLObjectType, GraphQLInterfaceTyp
         """Find the type at each VertexPath."""
         return get_types(self.metadata_table)
 
+    @cached_property
+    def classes_with_missing_counts(self) -> Set[str]:
+        """Return whether class counts are available for all vertices and edges used."""
```

Comment:
This seems imprecise: `whether` makes me expect a boolean, but the result seems to be the list of classes whose counts are missing. The boolean is perhaps `len(result) > 0`?

This sentence is valuable, as it describes the intent / scenario when I'd use this property. But it does not reflect the actual return value, so perhaps rephrase or switch to a long docstring.